week-3/Homework/mandatory/1-oxygen-levels.js

```diff
@@ -27,10 +27,29 @@ console.log(newdessert); // returns ice cream
 /* + + + + + + + */
 /* Now try the exercise */
 
-function safeLevels() {
+const oxygenLs1 = ["24.2%", "11.3%", "19.9%", "23.1%", "29.3%", "20.2%"];
+const oxygenLs2 = ["30.8%", "23.5%", "18.8%", "19.5%", "20.2%", "31.6%"];
+
+function safeLevels(strArr) {
+
+  let convertedArr = strArr.map( function (oxyLevel){
+    let sub = oxyLevel.substring(0,4);
+    return parseFloat(sub);
+    }
+  );
+
+  function findFirstPlanet(allLevels){
```

Comment:
```suggestion
  function findFirstPlanet(level){
```
Be careful with your parameter name on line 41 - you are using a plural but this parameter represents a **single** oxygen level.